src/js/background/background.js

```diff
@@ -1,5 +1,3 @@
-(async function () {
```

Comment:
Review note: As part of moving all the context menu code to its own file — wrapping the entire file in an async IFEE statement was no longer necessary.